Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -40,6 +46,23 @@ class GherkinState {
     
     fileprivate var missingStepsImplementations = [String]()
     
+    override init() {
+        super.init()
+        XCTestObservationCenter.shared.addTestObserver(self)
+    }
+
+    deinit {
+        XCTestObservationCenter.shared.removeTestObserver(self)
+    }
+
+    func testCase(_ testCase: XCTestCase, didFailWithDescription description: String, inFile filePath: String?, atLine lineNumber: Int) {
+        guard let test = self.test else { return }
+        let file = "\(test.state.file!)"
```

Comment:
While testing this to see how it affected the test results in Xcode I changed SanitySteps.swift:24 to be `XCTAssertTrue(false)` to force a fail, and it crashed on this line. For some reason the file is `nil`?